lib/licenses.nix

```diff
@@ -595,6 +595,11 @@ in mkLicense lset) ({
     url = "https://old.calculate-linux.org/packages/licenses/iASL";
   };
 
+  icu = {
+    spdxId = "ICU";
+    fullName = "ICU";
+  };
```

Comment:
If we don't already have it here I think ICU should be defined inline at `dps8m.meta`.

Reply:
I disagree, since it's a license that isn't specific to this project. (In fact the only reason it doesn't seem to be used for icu itself is that icu isn't marked with a license D:)

Reply:
In any case I don't care and I think licensing is a waste of time and thought.